symbol sets/src/ISOM2017_15000.xmap

```diff
@@ -0,0 +1,10029 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<map xmlns="http://openorienteering.org/apps/mapper/xml/v2" version="6">
+    <notes>ISOM 2017
+Symbols for OpenOrienteering Mapper
+Created by Jukka Liikari and Håvard Tveite, IOF Map Commission
```

Comment:
This looks like Jukka and Havard hold copyright and need to agree that OpenOrienteering may redistribute this file under its terms.